<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GGraphicsViewOptions</class>
 <widget class="QDialog" name="GGraphicsViewOptions">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>603</width>
    <height>692</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Configuration du graphique</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <widget class="QGroupBox" name="groupBox_4">
     <property name="title">
      <string> Couleurs et tailles </string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QWidget" name="widgetBackgroundColor" native="true">
        <layout class="QHBoxLayout" name="horizontalLayoutBackgroundColor">
         <property name="leftMargin">
          <number>0</number>
         </property>
         <property name="topMargin">
          <number>4</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>4</number>
         </property>
         <item>
          <widget class="QLabel" name="label">
           <property name="text">
            <string>Couleur d'arrière plan :</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QWidget" name="widgetSelectedColor" native="true">
        <layout class="QHBoxLayout" name="horizontalLayoutSelectedColor">
         <property name="leftMargin">
          <number>0</number>
         </property>
         <property name="topMargin">
          <number>4</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>4</number>
         </property>
         <item>
          <widget class="QLabel" name="label_2">
           <property name="text">
            <string>Couleur de sélection :</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QWidget" name="widget_3" native="true">
        <layout class="QHBoxLayout" name="horizontalLayout_3">
         <property name="leftMargin">
          <number>0</number>
         </property>
         <property name="topMargin">
          <number>4</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>4</number>
         </property>
         <item>
          <widget class="QLabel" name="label_3">
           <property name="text">
            <string>Taille des points :</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer_2">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QDoubleSpinBox" name="doubleSpinBoxPointSize">
           <property name="minimumSize">
            <size>
             <width>125</width>
             <height>0</height>
            </size>
           </property>
           <property name="decimals">
            <number>1</number>
           </property>
           <property name="minimum">
            <double>1.000000000000000</double>
           </property>
           <property name="maximum">
            <double>999.000000000000000</double>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_7">
           <property name="minimumSize">
            <size>
             <width>14</width>
             <height>0</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>14</width>
             <height>16777215</height>
            </size>
           </property>
           <property name="text">
            <string>px</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox_3">
     <property name="title">
      <string> Dessin </string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_4">
      <item>
       <widget class="QCheckBox" name="checkBoxDrawAxis">
        <property name="text">
         <string>Affichage des axes</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="checkBoxUseTransparency">
        <property name="text">
         <string>Activer la transparence</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="checkBoxUseLight">
        <property name="text">
         <string>Activer la lumière</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string> Optimisation </string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_5">
      <item>
       <widget class="QRadioButton" name="radioButtonFastestAlways">
        <property name="text">
         <string>Toujours utiliser l'optimisation</string>
        </property>
        <attribute name="buttonGroup">
         <string notr="true">buttonGroupFastest</string>
        </attribute>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="radioButtonFastestNormal">
        <property name="text">
         <string>Utiliser l'optimisation seulement pendant les déplacements</string>
        </property>
        <attribute name="buttonGroup">
         <string notr="true">buttonGroupFastest</string>
        </attribute>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="radioButtonFastestNever">
        <property name="text">
         <string>Ne jamais utiliser l'optimisation</string>
        </property>
        <attribute name="buttonGroup">
         <string notr="true">buttonGroupFastest</string>
        </attribute>
       </widget>
      </item>
      <item>
       <widget class="QWidget" name="widget" native="true">
        <layout class="QHBoxLayout" name="horizontalLayout">
         <property name="leftMargin">
          <number>0</number>
         </property>
         <property name="topMargin">
          <number>4</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>4</number>
         </property>
         <item>
          <widget class="QLabel" name="label_4">
           <property name="text">
            <string>Redessiner sans optimisation après :</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QSpinBox" name="spinBoxFastDrawTime">
           <property name="minimumSize">
            <size>
             <width>125</width>
             <height>0</height>
            </size>
           </property>
           <property name="maximum">
            <number>5000</number>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_5">
           <property name="minimumSize">
            <size>
             <width>14</width>
             <height>0</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>14</width>
             <height>16777215</height>
            </size>
           </property>
           <property name="text">
            <string>ms</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox_2">
     <property name="title">
      <string> Coordonnées de la caméra </string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <item>
       <widget class="QCheckBox" name="checkBoxCameraPosition">
        <property name="text">
         <string>Position</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="checkBoxCameraCenterCoordinates">
        <property name="text">
         <string>Centre de la vue</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="checkBoxCameraViewDirection">
        <property name="text">
         <string>Rotation</string>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QRadioButton" name="radioButtonUpperLeft">
          <property name="text">
           <string>Coin haut gauche</string>
          </property>
          <attribute name="buttonGroup">
           <string notr="true">buttonGroupCameraInfoPosition</string>
          </attribute>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="radioButtonUpperRight">
          <property name="text">
           <string>Coin haut droit</string>
          </property>
          <attribute name="buttonGroup">
           <string notr="true">buttonGroupCameraInfoPosition</string>
          </attribute>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="radioButtonLowerRight">
          <property name="text">
           <string>Coin bas droit</string>
          </property>
          <attribute name="buttonGroup">
           <string notr="true">buttonGroupCameraInfoPosition</string>
          </attribute>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="radioButtonLowerLeft">
          <property name="text">
           <string>Coin bas gauche</string>
          </property>
          <attribute name="buttonGroup">
           <string notr="true">buttonGroupCameraInfoPosition</string>
          </attribute>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox_5">
     <property name="title">
      <string> Vue </string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_6">
      <item>
       <widget class="QRadioButton" name="radioButtonPerspective">
        <property name="text">
         <string>Perspective</string>
        </property>
        <attribute name="buttonGroup">
         <string notr="true">buttonGroupViewType</string>
        </attribute>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="radioButtonOrthographic">
        <property name="text">
         <string>Orthographique</string>
        </property>
        <attribute name="buttonGroup">
         <string notr="true">buttonGroupViewType</string>
        </attribute>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <widget class="QWidget" name="widget_5" native="true">
     <layout class="QHBoxLayout" name="horizontalLayout_4">
      <item>
       <widget class="QPushButton" name="pushButtonSaveDefault">
        <property name="text">
         <string>Sauvegarder par défaut</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QDialogButtonBox" name="buttonBox">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="standardButtons">
         <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>GGraphicsViewOptions</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>GGraphicsViewOptions</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <buttongroups>
  <buttongroup name="buttonGroupCameraInfoPosition"/>
  <buttongroup name="buttonGroupViewType"/>
  <buttongroup name="buttonGroupFastest"/>
 </buttongroups>
</ui>
